packages/nuxi/src/utils/cjs.ts

```diff
@@ -15,7 +15,7 @@ export function getModulePaths (paths?: string | string[]): string[] {
     .filter(Boolean) as string[]
 }
 
-const _require = createRequire(process.cwd())
+const _require = createRequire(import.meta.url)
```

Comment:
Despite the name `cjs`, I can clearly see that Nuxt ships this chunk as a shared module imported in `cli.mjs` so I assume this is meant to run as ESM.

Let me know if this is a wrong assumption and I should use `__filename` here instead.